web/src/components/FormStateContacts.js

```diff
@@ -45,17 +52,52 @@ Contacts.propTypes = {
 
 const FormStateContacts = ({ handleSubmit, pristine, reset, submitting }) => (
   <form onSubmit={handleSubmit}>
-    <Field name="medicaidDirectorName" component={Input} label="Director" />
+    <SectionHeader>
+      We already have some information about Vermont from our records.
+    </SectionHeader>
+
+    <SectionHeader>Medicaid office:</SectionHeader>
+    <FormSection name="medicaidOffice">
+      <Field name="address1" component={Input} label="Address" />
+      <Field name="address2" component={Input} label="Address (continued)" />
+      <Flex wrap mx={-2}>
+        <Box p={2} w={[1, 1 / 2]}>
+          <Field name="city" component={Input} label="City" />
+        </Box>
+        <Box p={2} w={[1 / 2, 1 / 4]}>
+          <Field name="state" component={Input} label="State" />
+        </Box>
+        <Box p={2} w={[1 / 2, 1 / 4]}>
+          <Field name="zip" component={Input} label="Zip" />
+        </Box>
+      </Flex>
+    </FormSection>
+
+    <SectionHeader>Medicaid Director:</SectionHeader>
+    <FormSection name="medicaidDirector">
+      <Field name="name" component={Input} label="Name" />
+      <Field name="email" component={Input} label="Email address" />
+      <Field
+        name="phone"
+        type="tel"
+        pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}"
+        component={Input}
+        label="Phone number"
+      />
+    </FormSection>
+
     <FieldArray name="contacts" component={Contacts} />
-    <Divider my={4} color="gray2" />
-    <div>
-      <button type="submit" disabled={submitting}>
-        Submit
-      </button>
-      <button type="button" disabled={pristine || submitting} onClick={reset}>
-        Clear Values
-      </button>
-    </div>
+
+    {false && (
```

Comment:
This is essentially commenting out the submit/clear buttons, right?  I'm usually a stickler about deleting commented code rather than keeping it around.  Not going to block this PR over it, just letting you know how I do, for future's sake.  😉